src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -94,20 +103,27 @@ public class TxPool : ITxPool, IDisposable
 
             _headInfo.HeadChanged += OnHeadChange;
 
-            _filterPipeline.Add(new NullHashTxFilter());
-            _filterPipeline.Add(new AlreadyKnownTxFilter(_hashCache, _logger));
-            _filterPipeline.Add(new MalformedTxFilter(_specProvider, validator, _logger));
-            _filterPipeline.Add(new GasLimitTxFilter(_headInfo, txPoolConfig, _logger));
-            _filterPipeline.Add(new UnknownSenderFilter(ecdsa, _logger));
-            _filterPipeline.Add(new LowNonceFilter(_logger)); // has to be after UnknownSenderFilter as it uses sender
-            _filterPipeline.Add(new GapNonceFilter(_transactions, _logger));
-            _filterPipeline.Add(new TooExpensiveTxFilter(_transactions, _logger));
-            _filterPipeline.Add(new FeeTooLowFilter(_headInfo, _transactions, logManager));
-            if (incomingTxFilter is not null)
+            _filterGasLimitTx = new GasLimitTxFilter(_headInfo, txPoolConfig, _logger);
+            _filterMalformedTx = new MalformedTxFilter(_specProvider, validator, _logger);
+            _filterFeeTooLow = new FeeTooLowFilter(_headInfo, _transactions, _logger);
+            _filterNullHashTx = new NullHashTxFilter();
+            _filterAlreadyKnownTx = new AlreadyKnownTxFilter(_hashCache, _logger);
+            _filterUnknownSender = new UnknownSenderFilter(ecdsa, _logger);
+            _filterBalanceZero = new BalanceZeroFilter(_logger);
+            _filterBalanceTooLow = new BalanceTooLowFilter(_transactions, _logger);
+            _filterLowNonce = new LowNonceFilter(_logger); // has to be after UnknownSenderFilter as it uses sender
+            _filterGapNonce = new GapNonceFilter(_transactions, _logger);
+            _filterIncomingTx = incomingTxFilter;
+            _filterDeployedCode = new DeployedCodeFilter(_specProvider, _accounts);
+
+            int? reportMinutes = txPoolConfig.ReportMinutes;
+            if (reportMinutes.HasValue)
             {
-                _filterPipeline.Add(incomingTxFilter);
+                _timer = TimerFactory.Default.CreateTimer(TimeSpan.FromMinutes(reportMinutes.Value));
+                _timer.AutoReset = false;
+                _timer.Elapsed += TimerOnElapsed;
+                _timer.Start();
```

Comment:
If the log level isn't enough, don't start the timer.

Reply:
Done